Files/Helpers/NavigationHelpers.cs

```diff
@@ -123,7 +131,7 @@ public static async void OpenItemsWithExecutable(IShellPage associatedInstance,
         /// <param name="openSilent">Determines whether history of opened item is saved (... to Recent Items/Windows Timeline/opening in background)</param>
         /// <param name="openViaApplicationPicker">Determines whether open file using application picker</param>
         /// <param name="selectItems">List of filenames that are selected upon navigation</param>
```

Comment:
Could you add docs for `forceOpenInNewTab`. And also while at it, `args` too? @gave92